site/docs/components/AppDemo.tsx

```diff
@@ -0,0 +1,125 @@
+'use client';
+import { OnchainKitProvider } from '@coinbase/onchainkit';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import type { ReactNode } from 'react';
+import { http, WagmiProvider, createConfig } from 'wagmi';
+import { base, baseSepolia } from 'wagmi/chains';
+import { coinbaseWallet } from 'wagmi/connectors';
+import '@coinbase/onchainkit/styles.css';
+import { createWalletClient } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { useTheme } from '../contexts/Theme.tsx';
+
+const queryClient = new QueryClient();
+
+const demoWalletConnector = () => () => {
+  const demoWalletPK = import.meta.env.VITE_DEMO_WALLET_PK;
```

Comment:
This demo wallet will never have a value associated with it therefore I believe it's fine to leak the PK in the client.